packages/stark-ui/assets/stark-ui-bundle.scss

```diff
@@ -0,0 +1,49 @@
+/* Stark styles */
+@import "../assets/theming/base-theme";
+@import "../assets/styles/base";
```

Comment:
I think this line should be before the `base-theme` because it imports `normalize.css` which should be the first thing to be done before start defining rules. Actually, we define a different `margin` in the h1, h2. etc tags inside the `base-theme` that are then overwritten by `normalize.css` :s

And in the GETTING STARTED the examples have the right order ;)